modules/install-creating-install-config-aws-local-zones.adoc

```diff
@@ -3,33 +3,31 @@
 
 :_content-type: PROCEDURE
 [id="install-creating-install-config-aws-local-zones_{context}"]
-= Modifying an installation configuration file to use AWS Local Zones subnets
+= Modifying an installation configuration file to use AWS Local Zones
 
-Modify an `install-config.yaml` file to include AWS Local Zones subnets.
+Modify an `install-config.yaml` file to include AWS Local Zones.
 
 .Prerequisites
 
-* You created subnets by using the procedure "Creating a subnet in AWS Local Zones".
 * You created an `install-config.yaml` file by using the procedure "Creating the installation configuration file".
 
 .Procedure
 
-* Add the VPC and Local Zone subnets as the values of the `platform.aws.subnets` property. As an example:
+* Add Local Zones as values to the `platform.aws.szones` property. As an example:
 +
 [source,yaml]
 ----
 ...
 platform:
   aws:
     region: us-west-2
-    subnets: <1>
-    - publicSubnetId-1
-    - publicSubnetId-2
-    - publicSubnetId-3
-    - privateSubnetId-1
-    - privateSubnetId-2
-    - privateSubnetId-3
-    - publicSubnetId-LocalZone-1
+compute:
+- name: edge
+  platform:
+    aws:
+      zones: <1>
+      - <local_zone_name_1>
+      - <local_zone_name_2>
 ...
 ----
-<1> List of subnets created in the Availability and Local Zones.
\ No newline at end of file
+<1> List of Local Zones that associate with the defined AWS opt-in Region.
```

Comment:
It would be nice to add the `Next Steps` here pointing to `Deploying the cluster`